.github/workflows/publish.yaml

```diff
@@ -90,13 +94,11 @@ jobs:
       - name: Create GitHub release
         id: gh_release
         run: |
-          # Get the version file that we updated in the previous job
           wget https://raw.githubusercontent.com/ersilia-os/ersilia/master/ersilia/_static_version.py
         
           month=$(date +'%B')
           year=$(date +'%Y')
           title="$month $year"
-          # Extract the version using sed from _static_version.py
```

Comment:
Please don't delete comments that are there. 😅

Reply:
This is well noted. Thank you.